tests/test_makepackage.py

```diff
@@ -24,9 +24,7 @@ def run_cmds(cmds: list[cmd_command]):
             )
 
 
-def test_pkg_no_CLI(
```

Comment:
Did you run the tests in Python 3.11? The tests did pass in Python 3.9, but both failed in Python 3.11, with the same error:
<img width="498" alt="image" src="https://github.com/nyggus/makepackage/assets/39629775/db692495-bec7-4143-b2bf-f5784ece1644">